recipe/gen_patch_json.py

```diff
@@ -654,6 +654,12 @@ def _gen_new_index(repodata, subdir):
         if any(dep.startswith("zstd >=1.4") for dep in deps):
             _pin_looser(fn, record, "zstd", max_pin="x.x")
 
+        if any(dep.startswith("openmpi >=4.0") for dep in deps):
```

Comment:
@isuruf Wouldn't it be better if we just say 'openmpi>=4' here?

Reply:
Nope. We don't want to patch newer packages. Repodata patches should apply to  older ones and not new ones.

Reply:
But what happens to openmpi 4.1.0?